R/spark_gateway.R

```diff
@@ -34,7 +35,7 @@ gateway_connection <- function(master, config) {
 spark_gateway_connection <- function(master, config, gatewayInfo, gatewayAddress) {
   tryCatch({
     # set timeout for socket connection
-    timeout <- spark_config_value(config, "sparklyr.backend.timeout", 60)
+    timeout <- spark_config_value(config, "sparklyr.backend.timeout", 30 * 24 * 60 * 60)
```

Comment:
Yes, actually this was a fix performed over `sparkR`. Someone mentioned at Spark Summit SFO that they weren't using `sparkR` since it would timeout constantly for really long jobs, not 100% sure this was the main timeout we fixed, but I'm almost certain. So if anything, you might want to port this back to `sparkR`.